fhir-parent/pom.xml

```diff
@@ -529,6 +530,18 @@
                 <artifactId>logback-core</artifactId>
                 <version>1.2.3</version>
             </dependency>
+            <!-- 
+            Note, we can't use this until we upgrade the Open Liberty server
+            <dependency>
+                <groupId>jakarta.enterprise.concurrent</groupId>
+                <artifactId>jakarta.enterprise.concurrent-api</artifactId>
+                <version>2.0.0</version>
+            </dependency> -->
+            <dependency>
+                <groupId>javax.enterprise.concurrent</groupId>
+                <artifactId>javax.enterprise.concurrent-api</artifactId>
+                <version>1.1</version>
+            </dependency>
```

Comment:
is this actually used?   or did you pull that out when you simplified things?

Reply:
No - pulled out. great catch.